pycon/admin.py

```diff
@@ -136,6 +136,18 @@ class SponsorTutorialAdmin(ProposalMarkEditAdmin):
     ]
 
 
+class SpecialEventAdmin(MarkEditAdmin):
+    list_display = [
+        'name',
+        'start',
+        'end',
+        'location',
+        'published'
+    ]
+    ordering = ['-start']
+    prepopulated_fields = {"slug": ("name",)}
```

Comment:
A couple of nice additions to the admin: a published list filter, and a search box on name and description.